pkg/kapis/devops/v1alpha2/register.go

```diff
@@ -653,6 +653,16 @@ func AddPipelineToWebService(webservice *restful.WebService, devopsClient devops
 			Reads(devops.ReqJenkinsfile{}).
 			Returns(http.StatusOK, api.StatusOK, map[string]interface{}{}).
 			Writes(map[string]interface{}{}))
+
+		// download PipelineRun artifact
+		webservice.Route(webservice.GET("/devops/{devops}/pipelines/{pipeline}/artifact/runs/{run}").
```

Comment:
It's very good. I revised it.

Reply:
We can change the existing one.

Reply:
Do you want to change artifact list to other route？why not use `/devops/{devops}/pipelines/{pipeline}/runs/{run}/artifacts/download`, be more clarity?

Reply:
+1 from me. I agree with the new API `/devops/{devops}/pipelines/{pipeline}/runs/{run}/artifacts/download`.

BTW, the old API should also work properly.

Reply:
I misunderstood the API `/devops/{devops}/pipelines/{pipeline}/runs/{run}/artifacts`.

Currently, we didn't have an API to download the artifact files. So, we need an API for that. Got you.

+1 `/devops/{devops}/pipelines/{pipeline}/runs/{run}/artifacts/download`

Reply:
By the way, how about putting `/devops/{devops}/pipelines/{pipeline}/runs/{run}/artifacts/download` next by `/devops/{devops}/pipelines/{pipeline}/runs/{run}/artifacts`. It would be easier for use to lookup the similar APIs.